app/controllers/api/v1/orders_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Api::V1::OrdersController < ApplicationController
   def create
-    user = User.new(guest_params)
+    user = GuestCreator.create(params[:userParams])
```

Comment:
This allows for a clean interface to perform everything needed to create a guest record